providers/bind/bindProvider.go

```diff
@@ -107,12 +103,12 @@ func rrToRecord(rr dns.RR, origin string, replace_serial uint32) (models.RecordC
 
 func makeDefaultSOA(info SoaInfo, origin string) *models.RecordConfig {
 	// Make a default SOA record in case one isn't found:
-	soa_rec := models.RecordConfig{
+	soaRec := models.RecordConfig{
 		Type: "SOA",
 		Name: "@",
 	}
-	soa_rec.NameFQDN = dnsutil.AddOrigin(soa_rec.Name, origin)
-
+	soaRec.NameFQDN = dnsutil.AddOrigin(soaRec.Name, origin)
+	//TODO(cpeterson): are these sane defaults?
```

Comment:
@tlimoncelli I'm not sure if there is anything we can put for these defaults that makes more sense, but this still strikes me as odd.